### PR TITLE
Standardize BARTpho model card: badges, new examples, fixed broken im…

### PR DESCRIPTION
**Pull request by eshwanthkartitr**
Updated bartpho.md

# What does this PR do?

Fixes # (issue)
Standardize BARTpho model card

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#create-a-pull-request),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [ ] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

Please review this @stevhliu

### CODE REVIEW

**Comment by eshwanthkartitr** [comment]
Hi @stevhliu,

Thank you for your dedication to maintaining high-quality documentation across the repository. I've standardized the BARTpho model card with several key improvements:

**Key Changes:**
- Standardized model card badges for consistency with other model documentation
- Added new practical examples to help users better understand implementation
- Fixed broken links and images that were impacting user experience

I'd appreciate your feedback on whether any additional elements would enhance the documentation further - specifically, would an architecture diagram improve clarity for users? I'm also open to any other suggestions you might have to make this model card even more valuable for the community.

Looking forward to your review and any insights you can share!

Best regards

File: docs/source/en/model_doc/bartpho.md [hunk 1]
```diff
@@ -16,78 +16,124 @@ rendered properly in your Markdown viewer.
...
-">
+<img alt="Hugging Face Model" src="https://img.shields.io/badge/Model%20Hub-BARTpho-blue">
+<img alt="License" src="https://img.shields.io/badge/License-Apache%202.0-green">
+<img alt="Language" src="https://img.shields.io/badge/Language-Vietnamese-orange">
+</div>
```
Comment (stevhliu) [inline docs/source/en/model_doc/bartpho.md]: Don't need to add these extras

File: docs/source/en/model_doc/bartpho.md [hunk 2]
```diff
@@ -16,78 +16,124 @@ rendered properly in your Markdown viewer.
...
 </div>
 
-## Overview
+# BARTpho
 
-The BARTpho model was proposed in [BARTpho: Pre-trained Sequence-to-Sequence Models for Vietnamese](https://huggingface.co/papers/2109.09701) by Nguyen Luong Tran, Duong Minh Le and Dat Quoc Nguyen.
+[BARTpho](https://arxiv.org/abs/2109.09701) is the first large-scale, monolingual sequence-to-sequence model pre-trained exclusively for Vietnamese, developed by [VinAI Research](https://huggingface.co/vinai).
+It’s based on the **BART** denoising autoencoder architecture, with adaptations from **mBART**, and comes in two variants — **word** and **syllable** — to handle the unique way Vietnamese uses whitespace.
+Think of it like a supercharged summarizer and text generator that really “gets” Vietnamese — both at the word and syllable level.
```
Comment (stevhliu) [inline docs/source/en/model_doc/bartpho.md]:
```suggestion
[BARTpho](https://huggingface.co/papers/2109.09701) is a large-scale Vietnamese sequence-to-sequence model. It offers a word-based and syllable-based version. This model is built on the [BART](./bart) large architecture with its denoising pretraining.
```

File: docs/source/en/model_doc/bartpho.md [hunk 3]
```diff
@@ -16,78 +16,124 @@ rendered properly in your Markdown viewer.
...
 
-The abstract from the paper is the following:
+You can find all official checkpoints in the [BARTpho collection](https://huggingface.co/collections/vinai/bartpho-66f8a74775316eaa77d59969).
```
Comment (stevhliu) [inline docs/source/en/model_doc/bartpho.md]:
```suggestion
You can find all the original checkpoints under the [VinAI](https://huggingface.co/vinai/models?search=bartpho) organization.
```

File: docs/source/en/model_doc/bartpho.md [hunk 4]
```diff
@@ -16,78 +16,124 @@ rendered properly in your Markdown viewer.
...
 
-*We present BARTpho with two versions -- BARTpho_word and BARTpho_syllable -- the first public large-scale monolingual
-sequence-to-sequence models pre-trained for Vietnamese. Our BARTpho uses the "large" architecture and pre-training
-scheme of the sequence-to-sequence denoising model BART, thus especially suitable for generative NLP tasks. Experiments
-on a downstream task of Vietnamese text summarization show that in both automatic and human evaluations, our BARTpho
-outperforms the strong baseline mBART and improves the state-of-the-art. We release BARTpho to facilitate future
-research and applications of generative Vietnamese NLP tasks.*
+> \[!TIP]
+> This model was contributed by [VinAI Research](https://huggingface.co/vinai).
+> Check out the `bartpho-word` and `bartpho-syllable` variants in the right sidebar for examples of summarization, punctuation restoration, and capitalization restoration.
```
Comment (stevhliu) [inline docs/source/en/model_doc/bartpho.md]:
```suggestion
> [!TIP]
> This model was contributed by [dqnguyen](https://huggingface.co/dqnguyen).
> Check out the BARTpho in the right sidebar for examples of how to apply BART to different language tasks.
```

File: docs/source/en/model_doc/bartpho.md [hunk 5]
```diff
@@ -16,78 +16,124 @@ rendered properly in your Markdown viewer.
...
 
-This model was contributed by [dqnguyen](https://huggingface.co/dqnguyen). The original code can be found [here](https://github.com/VinAIResearch/BARTpho).
+The example below demonstrates how to run summarization with \[`pipeline`] or load the model via \[`AutoModel`].
```
Comment (stevhliu) [inline docs/source/en/model_doc/bartpho.md]:
```suggestion
The example below demonstrates how to summarize text with [`Pipeline`] or the [`AutoModel`] class.
```

File: docs/source/en/model_doc/bartpho.md [hunk 6]
```diff
@@ -16,78 +16,124 @@ rendered properly in your Markdown viewer.
...
 
-## Usage example
+<hfoptions id="usage">
+<hfoption id="Pipeline">
```
Comment (stevhliu) [inline docs/source/en/model_doc/bartpho.md:40]:
```py
import torch
from transformers import pipeline

pipeline = pipeline(
   task="summarization",
   model="vinai/bartpho-word",
   torch_dtype=torch.float16,
   device=0
)

text = """
Quang tổng hợp hay gọi tắt là quang hợp là quá trình thu nhận và chuyển hóa năng lượng ánh sáng Mặt trời của thực vật, 
tảo và một số vi khuẩn để tạo ra hợp chất hữu cơ phục vụ bản thân cũng như làm nguồn thức ăn cho hầu hết các sinh vật 
trên Trái Đất. Quang hợp trong thực vật thường liên quan đến chất tố diệp lục màu xanh lá cây và tạo ra oxy như một sản phẩm phụ
"""
pipeline(text)
```

File: docs/source/en/model_doc/bartpho.md [hunk 7]
```diff
@@ -16,78 +16,124 @@ rendered properly in your Markdown viewer.
...
+)
+
+ARTICLE = "BARTpho là mô_hình Xử_lý ngôn_ngữ tự_nhiên đơn_ngữ cho tiếng Việt..."
+summary = summarizer(ARTICLE, max_length=25, min_length=10, do_sample=False)
+print(summary[0]['summary_text'])
+```
 
->>> tokenizer = AutoTokenizer.from_pretrained("vinai/bartpho-syllable")
+</hfoption>
+<hfoption id="AutoModel">
```
Comment (stevhliu) [inline docs/source/en/model_doc/bartpho.md:64]:
```py
import torch
from transformers import BartForConditionalGeneration, AutoTokenizer

tokenizer = AutoTokenizer.from_pretrained(
    "vinai/bartpho-word",
)
model = BartForConditionalGeneration.from_pretrained(
    "vinai/bartpho-word",
    torch_dtype=torch.float16,
    device_map="auto",
)

text = """
Quang tổng hợp hay gọi tắt là quang hợp là quá trình thu nhận và chuyển hóa năng lượng ánh sáng Mặt trời của thực vật, 
tảo và một số vi khuẩn để tạo ra hợp chất hữu cơ phục vụ bản thân cũng như làm nguồn thức ăn cho hầu hết các sinh vật 
trên Trái Đất. Quang hợp trong thực vật thường liên quan đến chất tố diệp lục màu xanh lá cây và tạo ra oxy như một sản phẩm phụ
"""
inputs = tokenizer(text, return_tensors="pt").to("cuda")

outputs = model.generate(inputs["input_ids"], num_beams=2, min_length=0, max_length=20)
tokenizer.batch_decode(outputs, skip_special_tokens=True, clean_up_tokenization_spaces=False)[0]
```

File: docs/source/en/model_doc/bartpho.md [hunk 8]
```diff
@@ -16,78 +16,124 @@ rendered properly in your Markdown viewer.
...
+TXT = "Chúng_tôi là những nghiên_cứu_viên."
+inputs = tokenizer(TXT, return_tensors="pt")
+features = model(**inputs)
+print("Shape:", features.last_hidden_state.shape)
+```
 
->>> # With TensorFlow 2.0+:
->>> from transformers import TFAutoModel
+</hfoption>
+<hfoption id="transformers-cli">
```
Comment (stevhliu) [inline docs/source/en/model_doc/bartpho.md]:
```suggestion
<hfoption id="transformers CLI">
```

File: docs/source/en/model_doc/bartpho.md [hunk 9]
```diff
@@ -16,78 +16,124 @@ rendered properly in your Markdown viewer.
...
 
->>> bartpho = TFAutoModel.from_pretrained("vinai/bartpho-syllable")
->>> input_ids = tokenizer(line, return_tensors="tf")
->>> features = bartpho(**input_ids)
+```bash
+transformers-cli download vinai/bartpho-word
```
Comment (stevhliu) [inline docs/source/en/model_doc/bartpho.md]:
```bash
echo -e "Quang tổng hợp hay gọi tắt là quang hợp là quá trình thu nhận và chuyển hóa năng lượng ánh sáng Mặt trời của thực vật, 
tảo và một số vi khuẩn để tạo ra hợp chất hữu cơ phục vụ bản thân cũng như làm nguồn thức ăn cho hầu hết các sinh vật 
trên Trái Đất. Quang hợp trong thực vật thường liên quan đến chất tố diệp lục màu xanh lá cây và tạo ra oxy như một sản phẩm phụ" | transformers run --task summarization --model vinai/bartpho-word --device 0
```

File: docs/source/en/model_doc/bartpho.md [hunk 10]
```diff
@@ -16,78 +16,124 @@ rendered properly in your Markdown viewer.
...
 ```
 
-## Usage tips
+</hfoption>
+</hfoptions>
+
+Quantization reduces the memory footprint of large models by storing weights in lower precision. See the [BitsAndBytes Quantization guide](https://huggingface.co/docs/transformers/quantization/bitsandbytes) for details.
```
Comment (stevhliu) [inline docs/source/en/model_doc/bartpho.md]: Models not that big so we don't need a Quantization section

File: docs/source/en/model_doc/bartpho.md [hunk 11]
```diff
@@ -16,78 +16,124 @@ rendered properly in your Markdown viewer.
...
+quant_config = BitsAndBytesConfig(load_in_4bit=True, bnb_4bit_compute_dtype=torch.bfloat16)
+tokenizer = AutoTokenizer.from_pretrained("vinai/bartpho-word")
+model = AutoModel.from_pretrained("vinai/bartpho-word", quantization_config=quant_config, device_map="auto")
 ```
 
-- This implementation is only for tokenization: "monolingual_vocab_file" consists of Vietnamese-specialized types
-  extracted from the pre-trained SentencePiece model "vocab_file" that is available from the multilingual XLM-RoBERTa.
-  Other languages, if employing this pre-trained multilingual SentencePiece model "vocab_file" for subword
-  segmentation, can reuse BartphoTokenizer with their own language-specialized "monolingual_vocab_file".
+Use the [AttentionMaskVisualizer](https://github.com/huggingface/transformers/blob/main/src/transformers/utils/attention_visualizer.py) to see what tokens the model attends to.
```
Comment (stevhliu) [inline docs/source/en/model_doc/bartpho.md]: Not supported for this model yet

File: docs/source/en/model_doc/bartpho.md [hunk 12]
```diff
@@ -16,78 +16,124 @@ rendered properly in your Markdown viewer.
...
+TXT = "Chúng tôi là <mask> nghiên cứu viên."
+input_ids = tokenizer(TXT, return_tensors="pt")["input_ids"]
+logits = model(input_ids).logits
+masked_index = (input_ids == tokenizer.mask_token_id).nonzero().item()
+predicted_ids = torch.topk(logits[0, masked_index], 5).indices
+print(tokenizer.decode(predicted_ids, skip_special_tokens=True))
+```
 
-## BartphoTokenizer
+## Resources
```
Comment (stevhliu) [inline docs/source/en/model_doc/bartpho.md]: Remove this section

File: docs/source/en/model_doc/bartpho.md [hunk 13]
```diff
@@ -16,78 +16,124 @@ rendered properly in your Markdown viewer.
...
+model = MBartForConditionalGeneration.from_pretrained("vinai/bartpho-syllable")
+tokenizer = AutoTokenizer.from_pretrained("vinai/bartpho-syllable")
+
+TXT = "Chúng tôi là <mask> nghiên cứu viên."
+input_ids = tokenizer(TXT, return_tensors="pt")["input_ids"]
+logits = model(input_ids).logits
+masked_index = (input_ids == tokenizer.mask_token_id).nonzero().item()
+predicted_ids = torch.topk(logits[0, masked_index], 5).indices
+print(tokenizer.decode(predicted_ids, skip_special_tokens=True))
+```
```
Comment (stevhliu) [inline docs/source/en/model_doc/bartpho.md]:
```suggestion
- BARTphop uses the large architecture of BART with an additional layer-normalization layer on top of the encoder and decoder. The BART-specific classes should be replaced with the mBART-specific classes.
- This implementation only handles tokenization through the `monolingual_vocab_file` file. This is a Vietnamese-specific subset of token types taken from that multilingual vocabulary. If you want to use this tokenizer for another language, replace the `monolingual_vocab_file` with one specialized for your target language.
```

**Review by stevhliu — COMMENTED** [review]
Good start! Next time, make sure the code examples run and are aligned

File: docs/source/en/model_doc/bartpho.md (hunk 1 above)
Comment (eshwanthkartitr) [reply docs/source/en/model_doc/bartpho.md]: Removed the labels

Comment (stevhliu) [reply docs/source/en/model_doc/bartpho.md]: You only need to remove the ones I highlighted. The other labels (PyTorch, TensorFlow, Flax) are still valid

**Review by stevhliu — APPROVED** [review]
Thanks!
